src/SpeakDirectiveParser.js

```diff
@@ -37,12 +37,10 @@ export function extractAlexaTextResponse(alexaRawResponse) {
   try {
     avsDirective = JSON.parse(avsDirectiveBuffer);
   } catch (e) {
-    console.error(
-      "JSON parse error: " + util.inspect(e, { showHidden: true, depth: null })
-    );
     throw new IllegalArgumentError(
-      "Given directive couldn't be parsed to a JSON object. Input: " +
-        avsDirectiveBuffer.toString()
+      `Given directive couldn't be parsed to a JSON object. Input: " ${avsDirectiveBuffer.toString()}
+      StackTrace:
```

Comment:
Does the error actually give out a stack trace? Did you verify it? If it is not a stack trace, then the error message needs to be updated.

Reply:
Yep. Verified by putting this line before the throw statement and it does print the stack trace. 

```javascript
console.log(`Given directive couldn't be parsed to a JSON object. Input: " ${avsDirectiveBuffer.toString()}
      StackTrace:
      ${util.inspect(e, { showHidden: true, depth: null })}`);
```